MekHQ/src/mekhq/campaign/Campaign.java

```diff
@@ -429,6 +432,7 @@ public String getTitle() {
 
     public void setCalendar(GregorianCalendar c) {
         calendar = c;
+        today = LocalDate.of(c.get(Calendar.YEAR), c.get(Calendar.MONTH) + 1, c.get(Calendar.DAY_OF_MONTH));
```

Comment:
Why maintain two different dates? We already have the "calendar", so maybe just a convenience property to convert the calendar's state to a LocalDate instead?

Reply:
Ah, never mind, I read the PR description.